README.md

```diff
@@ -1,11 +1,57 @@
 # rebar3_edoc_extensions
-[![Hex.pm Version][hexpm version]][hexpm]
```

Comment:
I suppose it make sense to keep badges as before

Reply:
I added badges back.